components/apimgt/org.wso2.carbon.apimgt.rest.api.webserver/src/main/resources/webserver-api.yaml

```diff
@@ -90,8 +90,8 @@ paths:
         - $ref: '#/parameters/Accept'
         - $ref: '#/parameters/If-None-Match'
         - $ref: '#/parameters/If-Modified-Since'
-      produces:
-        - text/html; charset=utf-8
+#      produces:
+#        - text/html; charset=utf-8
```

Comment:
Any reason to keep the commented code ?

Reply:
Fixed with https://github.com/RenukaFernando/carbon-apimgt/commit/fe0833d6cecaf9b8d24701dc2be2157ffdf245b9